server/src/main/resources/com/thoughtworks/go/server/dao/maps/JobInstance.xml

```diff
@@ -324,10 +323,7 @@
             stages.name AS stageName,
             pipelines.counter AS pipelineCounter,
             stages.counter AS stageCounter,
-            stages.rerunOfCounter AS rerunOfCounter,
-            (
-                SELECT TIMESTAMPDIFF(MILLISECOND, stateChangeTime, bst.stateChangeTime) FROM buildStateTransitions WHERE buildid = builds.id AND currentState = 'Scheduled'
-            ) as duration
```

Comment:
I checked, apparently there is no `duration` on the POJO, so removing the `duration` here and above in line 310 seems reasonable.

Reply:
The duration is used for sorting on JobHistory columns: https://github.com/gocd/gocd/pull/7069/files#diff-eac6a96b60afa44709aca82f5fa7a416R299

Issue https://github.com/gocd/gocd/issues/4756

Reply:
Thanks. Bringing back that line.